tracing-core/src/lib.rs

```diff
@@ -298,25 +298,25 @@ pub use std::sync::Once;
 pub(crate) mod spin;
 
 pub mod callsite;
+pub mod collect;
 pub mod dispatcher;
```

Comment:
we should _probably_ keep these module names consistent: either `collector` and `dispatcher`, or `collect` and `dispatch`. i could go either way on which one, but we should make a decision: i'd prefer not to merge this PR with the module named `collect` and then immediately changing it again. if we rename the `dispatcher` module to `dispatch`, it would be fine to do that in a follow-up.

Reply:
My two cents: `collector`/`dispatcher`. Even though the traits are called `Collect` and `Dispatch`, the containing modules are what one would use to write code about collectors and dispatchers. 
(Note: I'm not a native speaker and both are fine. I agree on consistency being key though.)